src/Traits/HelperTestCaseTrait.php

```
@@ -105,4 +105,29 @@ public static function assertMethodExists($method, $class, $msg = '')
 
         static::assertTrue(method_exists($class, $method), $msg);
     }
+
+    /**
+     * Assert that the element exists in the json.
+     *
+     * @param mixed  $needle
+     * @param array  $haystack
+     * @param string $msg
+     *
+     * @return void
+     */
+    public static function assertInJson($needle, $haystack, $msg = '')
+    {
+        if ($msg === '') {
+            $msg = "Expected the array to contain the element '$needle'.";
+        }
+
+        $array = json_decode($needle, true);
+
+        if (json_last_error() !== JSON_ERROR_NONE) {
```

maybe better to actually throw an exception here rather than failing the assertion